apcd-cms/src/apps/registrations/templates/submission_form/submission_form.html

```diff
@@ -895,46 +539,34 @@ <h6>
               <input
                 type="number"
                 name="claims_encounters_volume"
-                required=""
+                required
                 class="integerfield"
                 id="claims_encounters_volume"
                 inputmode="numeric"
                 min="1"
               />
-
-              <div id="help-text-claims_encounters_volume" class="help-text">
-                <!-- More HTML5 attributes are added via JavaScript based on the "Custom Field Name" value. -->
-              </div>
             </div>
 
-            <div
-              class="field-wrapper numberinput required js-prepend--field-wrapper"
-              id="total_claims_value--wrapper"
-            >
+            <div class="field-wrapper numberinput required s-affixed-input-wrapper">
               <div class="field-errors" style="display: none"></div>
 
               <label for="total_claims_value">
                 Total Claims Value (USD<sup>7</sup>)<span class="asterisk">*</span>
               </label>
 
-              <span class="js-prepend--span" data-text="$">$</span
-              ><input
+              <span class="s-affixed-input-wrapper__prefix">$</span>
```

Comment:
The `$` sign is appearing on it's own line for me:
![Screen Shot 2022-10-31 at 1 45 06 PM](https://user-images.githubusercontent.com/29575979/199085611-000b0492-516c-4b57-9d77-198e1981de71.png)

Reply:
Yes, thanks. I forgot I had snippets locally.

Please pull and try again. Same test.

No need for snippets locally as of [ec11137](https://github.com/TACC/Core-CMS-Custom/pull/35/commits/ec1113713519c5a60cae043d83453b6e9bc508b1).